src/vs/editor/contrib/codeAction/browser/codeActionMenu.ts

```diff
@@ -391,7 +394,10 @@ export class CodeActionMenu extends Disposable implements IEditorContribution {
 	}
 
 	public onEnterSet() {
-		this.codeActionList.value?.setSelection([this.currSelectedItem]);
+		// 0 case when first enabled item is the 0th element in list.
+		if (this.currSelectedItem === 0 || this.currSelectedItem) {
```

Comment:
Here it would be clearer to check if it is a number: `typeof this.currSelectedItem === 'number'`